src/main/java/sa/assertj/iterables/AreAtMost.java

```diff
@@ -0,0 +1,34 @@
+package sa.assertj.iterables;
+
+import org.assertj.core.api.Condition;
+import sa.assertj.Experiment;
+
+import java.util.ArrayList;
+import java.util.Random;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class AreAtMost extends Experiment {
+    static Experiment.DataProvider provider = (size, numOfSamples) -> {
+        Object[][] result = new Object[numOfSamples][];
+        for (int i = 0; i < numOfSamples; i++) {
+            ArrayList<Integer> list = new ArrayList<>();
+            Random rand = new Random();
+            list.add(1);
+            for (int j = 1; j < size; j++) {
+                list.add(rand.nextInt());
+            }
+
+            Condition<Integer> condition = new Condition<>(s -> s.equals(0), "equals zero");
+
+            result[i] = new Object[] { list, size, condition};
```

Comment:
I think you also have this in the other files, but it's not a #1 priority.

```suggestion
            result[i] = new Object[] { list, size, condition };
```